packages/cli/src/deploy/getResourceIds.ts

```diff
@@ -15,15 +17,25 @@ export async function getResourceIds({
   // TODO: PR to viem's getLogs to accept topics array so we can filter on all store events and quickly recreate this table's current state
 
   debug("looking up resource IDs for", worldDeploy.address);
-  const logs = await getLogs(client, {
-    strict: true,
-    address: worldDeploy.address,
-    fromBlock: worldDeploy.deployBlock,
-    toBlock: worldDeploy.stateBlock,
-    event: parseAbiItem(storeSpliceStaticDataEvent),
-    args: { tableId: storeTables.store_ResourceIds.tableId },
-  });
-
+  const logs = await pRetry(
+    () =>
+      getLogs(client, {
+        strict: true,
+        address: worldDeploy.address,
+        fromBlock: worldDeploy.deployBlock,
+        toBlock: worldDeploy.stateBlock,
+        event: parseAbiItem(storeSpliceStaticDataEvent),
+        args: { tableId: storeTables.store_ResourceIds.tableId },
+      }),
+    {
+      retries: 3,
+      onFailedAttempt: async (error) => {
+        const delay = error.attemptNumber * 500;
+        debug(`failed to get logs, retrying in ${delay}ms...`);
+        await wait(delay);
```

Comment:
no need for this here, I believe `pRetry` has its own back off internally
(I suspect this was copied from other spots where we also don't need to do this)

Reply:
Oh, yes just copied other examples :D